digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1417,7 +1381,6 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
     val linkIds = projectLinks.map(_.linkId).toSet ++ expiringRoadAddresses.values.map(_.linkId).toSet
     val roadLinks = (roadLinkService.fetchVVHRoadlinks(linkIds, frozenTimeVVHAPIServiceEnabled) ++
```

Comment:
we should probably remove these fetches?

Reply:
to my understanding we were suppose to add project_link geometry as is, unless i am not mistaken we now fetch new geometry from vvh and then push project to roadaddress table.

Reply:
We are able to do a ProjectLink -> RoadAddress conversion without using VVH at all. If the geometry has been updated since the change table does the conversion without problems.